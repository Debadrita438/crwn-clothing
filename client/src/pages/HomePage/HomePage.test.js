import { shallow } from 'enzyme';

import HomePage from './HomePage';

it('should render HomePage', () => {
    expect(shallow(<HomePage />)).toMatchSnapshot();
});