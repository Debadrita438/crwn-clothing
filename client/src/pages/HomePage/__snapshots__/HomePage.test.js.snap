// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage 1`] = `ShallowWrapper {}`;
